inference-engine/src/auto_plugin/auto_plugin.cpp

```diff
@@ -45,6 +45,8 @@ namespace {
     }
 }  // namespace
 
+thread_local ConfigType AutoInferencePlugin::_autoConfig = {};
```

Comment:
why we want it to be therad-local (and not just a class member)

Reply:
has removed.